config/redirects.txt

```diff
@@ -109,11 +108,9 @@
 ^\/deployment\/build-optimisation\/?(\?.+)?(#.+)?$ -> /learn/deploy/build-optimization/$1$2
 ^\/reference\/data-api\/?(\?.+)?(#.+)?$ -> /reference/api/overview/$1$2
 ^\/reference\/resinOS\/?(\?.+)?(#.+)?$ -> /reference/OS/$1
-^\/reference\/OS\/meta-balena\/?(\?.+)?(#.+)?$ -> /reference/OS/customer-board-support/$1$2
-^\/reference\/OS\/custom-device-support\/?(\?.+)?(#.+)?$ -> /reference/OS/customer-board-support/$1$2
+^\/reference\/OS\/meta-balena\/?(\?.+)?(#.+)?$ -> /reference/OS/custom-device-support/$1$2
 ^\/reference\/base-images\/resin-base-images\/?(\?.+)?(#.+)?$ -> /reference/base-images/base-images/$1$2
 ^\/reference\/OS\/overview\/1.x\/?(\?.+)?(#.+)?$ -> /reference/OS/overview/$1$2
 ^\/reference\/OS\/overview\/2.x\/?(\?.+)?(#.+)?$ -> /reference/OS/overview/$1$2
 ^\/reference\/OS\/network\/1.x\/?(\?.+)?(#.+)?$ -> /reference/OS/network/$1$2
 ^\/reference\/OS\/network\/2.x\/?(\?.+)?(#.+)?$ -> /reference/OS/network/$1$2
-^\/learn\/manage\/app-types\/?(\?.+)?(#.+)?$ -> /learn/manage/fleet-types/$1$2
```

Comment:
Again, not sure why we are deleting redirects

Reply:
More sloppy conflict resolution.